src/model/fxQuotes.js

```diff
@@ -65,13 +66,88 @@ class FxQuotesModel {
     }
   }
 
+  async checkDuplicateFxQuoteRequest (fxQuoteRequest) {
+    try {
+      // calculate a SHA-256 of the request
+      const hash = calculateRequestHash(fxQuoteRequest)
+      this.log.debug(`Calculated sha256 hash of quote request with id ${fxQuoteRequest.conversionRequestId} as: ${hash}`)
+
+      const dupchk = await this.db.getFxQuoteDuplicateCheck(fxQuoteRequest.conversionRequestId)
+      this.log.debug(`DB query for quote duplicate check with id ${fxQuoteRequest.conversionRequestId} returned: ${util.inspect(dupchk)}`)
+
+      if (!dupchk) {
+        // no existing record for this quoteId found
```

Comment:
Personally, I prefer add log instead of such comment:
```
this.log.info('no existing record for this quoteId found')
```
so we'll have additional info in logs during debugging

Reply:
done